src/Nordea/Components/DropdownFilter.elm

```diff
@@ -175,7 +183,10 @@ inputSearchView hasFocus searchString onInput onFocus =
                     [ Css.outline Css.none
                     , Css.border3 (Css.px 1) Css.solid Colors.blueNordea
                     ]
-                , Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                , if hasFocus then
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 0) (Css.px 0)
+                  else
+                    Css.borderRadius4 (Css.px 4) (Css.px 4) (Css.px 4) (Css.px 4)
```

Comment:
Would be great to convert the px's to rem's, then everything scales accordingly 😀

Reply:
Good point! Everything here should probably be converted then